FirebaseAuth/Sources/Public/FirebaseAuth/FIRRecaptchaBridge.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <TargetConditionals.h>
+
+#if TARGET_OS_IOS
+
+typedef void (^FIRAuthRecaptchaTokenCallback)(NSString *_Nonnull token,
+                                              NSError *_Nullable error,
+                                              BOOL linked);
```

Comment:
This is totally optional, but it may be more maintainable to mirror the `Result<String, Error>` pattern here.

```
typedef void (^FIRAuthRecaptchaTokenCallback)(NSString *_Nullable token,
                                              NSError *_Nullable error);
```
Where the `linked=NO` is instead handled by a thrown NSError (or maybe even `nil, nil`). 

This being said, it currently LGTM if you'd prefer to keep it as is.

Reply:
I purposely decided not to do this to keep the bridge to C and Objective C as explicit and small as possible.